### DISCUSSION TITLE
How to set cursor to pointer on specific cells?

### DISCUSSION THREAD

**Discussion by city17** [glideapps/glide-data-grid#418] [Q&A] [answered]
Hi, I've set up my datagrid so the very first column is a link icon that links to a specific page. But I'd like to have the cursor change to `pointer` when hovering over this.

<img width="86" alt="Schermafbeelding 2022-06-29 om 10 05 30" src="https://user-images.githubusercontent.com/13391662/176385073-de282c73-170f-493d-ac61-6f4a2155c36f.png">

Currently I've got it set up like this:

```jsx
  const handleItemHover = (args) => {
    if (args.kind === 'cell' && args.location[0] === 0 ) {
      document.getElementById('asset-grid').style.cursor = 'pointer'; // Potential implementation
      console.log('hovering over first column')
    }
  }
  
  <DataEditor onItemHovered={handleItemHover} />
```

But it seems that the Data Grid overrules these style changes. Is there a way to enable a specific cursor on certain cells? Thanks!

**Comment by jassmith** [comment, marked as answer]
This is now possible to set on a cell by cell basis in 5.0.0 :)

**Reply by city17** [reply]
Cool, thanks. How exactly would you do this in 5.0.0? 

I saw you changed from CSS-in-JS to changing CSS classes, but it wasn't clear to me how you'd use this new system to add hover effects (pointer) to a specific column of cells. Or is this something that will be implemented later?

**Comment by jassmith** [comment]
Sorry in 5.0.0 the method to do this is to set `cursor` prop on your GridCell. It overrides the cursor.

**Reply by city17** [reply]
Thanks! When do you expect that 5.0.0 will be the new stable released version?

**Comment by jassmith** [comment]
1 to 4 weeks.

**Comment by jassmith** [comment]
There are 4.99.0-alphaX packages out now you can play with. They are pretty much production ready if you don't have any problems with transpilation (we moved to esbuild).
